tests/parsing/errors/scanner/__snapshots__/parse.spec.js.snap

```diff
@@ -165,11 +165,11 @@ exports[`unclosedComment.js 1`] = `
 
 exports[`unclosedString.js 1`] = `
 "=====Parsetree==========================================
-let z = \\"eof\\"
+let z = \\"eof\\\\n\\"
```

Comment:
This is a natural change but one can argue both ways; whether to include the newline into the highlight or not

Reply:
How does this look in the terminal?

Reply:
Before:
<img width="740" alt="Screen Shot 2021-03-16 at 12 22 00 AM" src="https://user-images.githubusercontent.com/1909539/111270863-a2ccb480-85ed-11eb-80e4-fd8bf24acb0c.png">

After:
<img width="730" alt="Screen Shot 2021-03-16 at 12 22 38 AM" src="https://user-images.githubusercontent.com/1909539/111270928-b9730b80-85ed-11eb-867a-ec72a941620d.png">

No visible difference visually for the existing highlighter

Reply:
The newline might be an artefact from my editor. Everything before the end-of-file token should be a part.